roles/junit2json/tasks/merge.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file merge.yml for role redhatci.ocp.junit2json
+# merges multiple XMLs into 1 and converts it into JSON
+
+- name: Merge multiple JUnit XML files into single consolidated report
+  ansible.builtin.command:
+    cmd: |
+      junitparser merge "{{ junit2json_reports_list | join('" "') }}" -
+  register: _junit2json_merged_xml
+  changed_when:
+    - true
+  failed_when:
+    - _junit2json_merged_xml.rc != 0
+
+- name: Write merge resulting file
+  ansible.builtin.copy:
+    content: "{{ _junit2json_merged_xml.stdout }}"
+    dest: "{{ junit2json_output_dir }}/{{ junit2json_input_merged_report }}" # so that our findings don't find it
+    mode: '0644'
+  when:
+    - _junit2json_merged_xml.stdout | length > 10
```

Comment:
Why this 10 here? What's the meaning?
If that's a constant value, maybe this should be defined somewhere else and use it accordingly.